plyer/facades/volume.py

```diff
@@ -0,0 +1,35 @@
+'''
+Volume
+=======
+
+Simple Example
+---------------
+To set the volume::
+	>>> import plyer
+	>>> plyer.volume.status
```

Comment:
So you are accessing a ***`status`*** to ***set*** the volume via ***`_get_state()`*** method? I mean... why?